### PR TITLE
Multi color scheme

### PR DESCRIPTION
**Pull request by Naramsim**

### CODE REVIEW

**Comment by Naramsim** [comment]
@sarbbottam

File: src/change-color.js [hunk 1]
```diff
@@ -5,22 +5,31 @@ function updateStyle(selector, property, value) {
...
+    if (color && (color in colorScheme)) {
+        Object.keys(colorScheme[color]).forEach(key => {
+            updateStyle(`rect[fill='#${key}']`, 'fill', `#${colorScheme[color][key]}`);
+            updateStyle(`li[style='background-color: #${key}']`, 'backgroundColor', `#${colorScheme[color][key]}`);
         });
     }
 }
 
-module.exports = changeColor;
+module.exports = changeContributionsColor;
```
Comment (sarbbottam) [inline src/change-color.js:36]: Wow, this name is better! 👍 more descriptive,

File: src/change-color.js [hunk 2]
```diff
@@ -5,22 +5,31 @@ function updateStyle(selector, property, value) {
...
+            '44a340': 'EF6C00',
+            '1e6823': '000000'
+        }
     };
 
-    if (color) {
-        Object.keys(colorsHash).forEach(key => {
-            updateStyle(`rect[fill='#${key}']`, 'fill', `#${colorsHash[key]}`);
-            updateStyle(`li[style='background-color: #${key}']`, 'backgroundColor', `#${colorsHash[key]}`);
+    if (color && (color in colorScheme)) {
```
Comment (sarbbottam) [inline src/change-color.js]: I would use `colorScheme[color]` to verify existence.

File: src/change-color.js [hunk 3]
```diff
@@ -5,22 +5,31 @@ function updateStyle(selector, property, value) {
...
+        Object.keys(colorScheme[color]).forEach(key => {
```
Comment (sarbbottam) [inline src/change-color.js]: Cache `colorScheme[color]` as it is being used more than once.

File: src/change-color.js [hunk 4]
```diff
@@ -5,22 +5,31 @@ function updateStyle(selector, property, value) {
...
-    const color = localStorage.getItem('blueSky');
-    const colorsHash = {
-        'eeeeee': 'eeeeee',
-        'd6e685': '9DC1F6',
-        '8cc665': '629DF3',
-        '44a340': '2B7BF2',
-        '1e6823': '0363F0'
+function changeContributionsColor() {
+    const color = localStorage.getItem('color-scheme');
+    const colorScheme = {
```
Comment (sarbbottam) [inline src/change-color.js]: `colorSchemes` please note the `s`

File: src/change-color.js [hunk 5]
```diff
@@ -5,22 +5,31 @@ function updateStyle(selector, property, value) {
...
-function changeColor() {
-    const color = localStorage.getItem('blueSky');
-    const colorsHash = {
-        'eeeeee': 'eeeeee',
-        'd6e685': '9DC1F6',
-        '8cc665': '629DF3',
-        '44a340': '2B7BF2',
-        '1e6823': '0363F0'
+function changeContributionsColor() {
+    const color = localStorage.getItem('color-scheme');
```
Comment (sarbbottam) [inline src/change-color.js]: `colorScheme` please.

File: src/change-color.js (hunk 4 above)
Comment (sarbbottam) [inline src/change-color.js]: and the corresponding changes below.

**Comment by sarbbottam** [comment]
Please update `readme`.

instead of 

> localStorage.setItem("blueSky", 1) then reload

use 

> localStorage.setItem("color-scheme", "blue-sky") then reload

**Comment by sarbbottam** [comment]
Sorry for nit-picks, it is from maintainability perspective.

**Pull request merged**